src/components/Header/HeaderButton.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+
+import { Icon, Props as IconProps } from '../Icon'
+
+export interface HeaderButtonProps {
+  children?: React.ReactNode
+  url?: string
+  target?: string
+  icon?: IconProps['name']
+}
+
+const HeaderButtonComponent: React.FC<HeaderButtonProps & InjectedProps> = ({
+  theme,
+  ...props
+}) => (
+  <Wrapper theme={theme} href={props.url} target={props.target && props.target}>
+    {props.icon && (
+      <HeaderButtonIcon theme={theme} role="presentation">
+        <Icon name={props.icon}></Icon>
+      </HeaderButtonIcon>
+    )}
+    {props.children && props.children}
+  </Wrapper>
+)
+
+export const HeaderButton = withTheme(HeaderButtonComponent)
```

Comment:
[MUST] Is HeaderButton used for anything other than Header? If not, it should be included in the Header.
I want the components to be placed directly under 'components' only for user use.

Reply:
HeaderButton component already moved to Header directory. 👍